includes/admin/settings/class-settings-general.php

```diff
@@ -228,13 +228,20 @@ public function get_settings() {
 							'type'    => 'select',
 							'options' => give_get_country_list(),
 						),
-                        array(
-                            'name'  => esc_html__( 'General Options Docs Link', 'give' ),
-                            'id'    => 'general_options_docs_link',
-                            'url'   => esc_url( 'http://docs.givewp.com/settings-general' ),
-                            'title' => __( 'General Options', 'give' ),
-                            'type'  => 'give_docs_link',
-                        ),
+						array(
+							'name'    => esc_html__( 'Default State/Province', 'give' ),
+							'desc'    => esc_html__( 'The state/province your site operates from.', 'give' ),
+							'id'      => 'base_state',
+							'type'    => 'select',
+							'options' => give_states_list()[ get_option( 'give_settings' )['base_country'] ],
```

Comment:
@Sidsector9 Give support PHP 5.3+. Did you test with minimum supported version (array short tag will not work in that version)?

Reply:
@ravinderk I slept before checking the Travis build completed :sweat_smile: